shared/login/signup/passphrase/index.js

```diff
@@ -32,13 +32,15 @@ class Passphrase extends React.Component<Props, State> {
           type="password"
           errorText={this.props.passphrase === this.state.pass1 ? this.props.error : ''}
           onChangeText={pass1 => this.setState({pass1})}
+          uncontrolled={true}
```

Comment:
is the `value` prop needed on these?

Reply:
its ignored, i wanted to change it minimally. my hope is that when we can move to 56/57 we can go back to being controlled

Reply:
@chrisnojima sounds good

Reply:
@chrisnojima any particular reason we want it to be controlled? doesn't uncontrolled work better?

Reply:
i think in general we want controlled. its the 'react' way to do things, we have the props and its either in state or the store and we can do stuff to it, like reset it and etc. otherwise the state isn't really captured by 'our system' and it in the dom/rn tree somewhere. if we need it for performance reasons in some places thats fine but i think its usually simpler to reason about if its more like everything else (we have no other uncontrolled controls at all)

Reply:
Okay, I see. One correction: we do have one other uncontrolled control -- chat input

Reply:
(oh, also paper key input, for some reason)

Reply:
i don't mean this is the only instance of an uncontrolled input, i mean we don't have any other uncontrolled components (checkboxes, sliders, etc) only Input